src/Popover.js

```diff
@@ -1,79 +1,171 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
-import TetherContent from './TetherContent';
-import { getTetherAttachments, mapToCssModules, omit, tetherAttachements } from './utils';
+import PopperJS from 'popper.js';
+import PopperContent from './PopperContent';
+import { getTarget, DOMElement, mapToCssModules, omit } from './utils';
 
 const propTypes = {
-  placement: PropTypes.oneOf(tetherAttachements),
-  target: PropTypes.string.isRequired,
+  placement: PropTypes.oneOf(PopperJS.placements),
```

Comment:
@TheSharpieOne looks like the `Popover` component is missing the `container` prop which is in the v5 docs.